content/blog/2016/2016-06-20-all-in-jenkins-world.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "Go ALL IN for Jenkins World"
```

Comment:
I didn't notice that, I'll tweak before I publish this morning. I told @hinman55 I'd tackle this today